services/movie_session.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall=cinema_hall_id,
```

Comment:
This line should use the `cinema_hall` object instead of `cinema_hall_id`. You need to fetch the `CinemaHall` object using the provided `cinema_hall_id` before creating the `MovieSession`. This is to avoid unnecessary database loading as per the checklist.